haystack/document_store/faiss.py

```diff
@@ -242,30 +262,35 @@ def get_all_documents_generator(
         :param return_embedding: Whether to return the document embeddings.
         :param batch_size: When working with large number of documents, batching can help reduce memory footprint.
         """
+        index = index or self.index
         documents = super(FAISSDocumentStore, self).get_all_documents_generator(
-            index=index, filters=filters, batch_size=batch_size
+            index=index, filters=filters, batch_size=batch_size, return_embedding=False,
```

Comment:
Why do we hardcode `return_embeddings=False` here?

Reply:
This call goes to the `SQLDocumentStore`, which currently does not store embeddings.